mandatory/2-logic-error.js

```diff
@@ -1,16 +1,19 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  
+  return word.trim();
 }
-
-function getStringLength(word) {
-  return "word".length();
+word = "  CodeYourFuture  ";
+word = " CodeYourFuture teaches coding     ";
```

Comment:
You did not need to assign word in lines 7 and 8, because work parameter is coming from the test below. So all you need is to handle what's inside the function.

At the moment lines 7 & 8 are useless because the function is not picking the "word" Parameter from lines 7 &8 but it is picking from the argument that's been passed from below in the tests